### PR TITLE
Init offsets when initialOffset is latest in KafkaSource

### PR DESCRIPTION
**Pull request by pierDipi**
As per title.

Part of #312

## Proposed Changes

- Init offsets when initialOffset is latest in KafkaSource

**Release Note**

```release-note
None
```

**Docs**

```
None
```

### CODE REVIEW

File: control-plane/pkg/reconciler/base/receiver_condition_set.go [hunk 1]
```diff
@@ -250,3 +252,16 @@ func (manager *StatusConditionManager) TopicsNotPresentOrInvalid(topics []string
 	)
 	return fmt.Errorf("topics %v not present or invalid: check topic configuration", topics)
 }
+
+func (manager *StatusConditionManager) InitialOffsetNotCommitted(err error) error {
+	manager.Object.GetConditionSet().Manage(manager.Object.GetStatus()).MarkFalse(
+		ConditionInitialOffsetsCommitted,
+		"InitialOffsetNotCommitted",
```
Comment (pierDipi) [inline control-plane/pkg/reconciler/base/receiver_condition_set.go:259]:
```suggestion
		"InitialOffsetsNotCommitted",
```

File: control-plane/pkg/reconciler/kafka/testing/MockKafkaClient.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
+/*
```
Comment (aliok) [inline control-plane/pkg/reconciler/kafka/testing/MockKafkaClient.go:1]: File naming is not consistent with others here

**Comment by aliok** [comment]
/lgtm

I only have a comment about a file's name.

File: control-plane/pkg/reconciler/kafka/testing/MockKafkaClient.go (hunk 2 above)
Comment (pierDipi) [reply control-plane/pkg/reconciler/kafka/testing/MockKafkaClient.go:1]: True

/hold

Comment (pierDipi) [reply control-plane/pkg/reconciler/kafka/testing/MockKafkaClient.go]: Done
/unhold

**Review by aliok — APPROVED** [review]
/lgtm
/approve

**Pull request merged**